pulumi/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,10 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$bindir   = Join-Path -Path $toolsDir -ChildPath "bin"
+
+Uninstall-ChocolateyPath $bindir 'Machine'
```

Comment:
`Uninstall-ChocolateyPath` is not a default Chocolatey commandlet. You'll need to add this to your package.
An example can be found here: https://github.com/mkevenaar/chocolatey-packages/blob/master/automatic/mariadb.install/tools/Uninstall-ChocolateyPath.psm1
Also the module should be imported
```suggestion
$unPath = Join-Path $toolsPath 'Uninstall-ChocolateyPath.psm1'
Import-Module $unPath
Uninstall-ChocolateyPath $bindir 'Machine'
```